docs/workforce-management-framework/topics/pro-keycloak-enablement.adoc

```diff
@@ -94,20 +94,8 @@ NOTE: The `keycloakConfig` field is partially implemented in link:{WFM-RC-CoreUR
 +
 . At this point, the _Routes_ are protected by *{WFM-RC-KeycloakName}*.
 
-
-[id=pro-logging-in-to-the-keycloak-admin-console-{chapter}]
+[id={context}-accessing-the-keycloak-admin-console]
 [discrete]
-=== Logging in to the {WFM-RC-KeycloakName} Admin Console
-
-. Navigate to _http://localhost:8080/auth/_
+== Accessing the {WFM-RC-KeycloakName} Admin Console
 
-. Login as the _admin_ user using the following credentials:
```

Comment:
This was moved to the reference section and the reference section will appear directly below this.
Directly after this content is rendered, I have the following text:

"For login details, refer to the xref:{context}-credentials-for-the-demo-application[Credentials for the {WFM-RC-KeycloakName} Admin Console] section of xref:{context}-ref-demo-app[{WFM-RC-KeycloakName} Reference Material]."

Trust me on this change. Render this PR for a more clear picture. If you are still not happy with this. lets talk it through.